tasks/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+"""Consolidates invoke tasks and utilities
+
+"""
+from .tasks import *  # noqa
```

Comment:
Is that a good approach? Is there anyplace that we import tasks.py? The only place I see a potential is in archiver/listeners.py